internal/services/compute/orchestrated_virtual_machine_scale_set_resource.go

```diff
@@ -266,6 +266,8 @@ func resourceOrchestratedVirtualMachineScaleSet() *pluginsdk.Resource {
 			},
 
 			"priority_mix": OrchestratedVirtualMachineScaleSetPriorityMixPolicySchema(),
+
+			"additional_unattend_content": additionalUnattendContentSchema(),
```

Comment:
I don't think this should be a top level property since it only applies to Windows VMs.

This property was actually preemptively added quite a while ago but commented out because we hadn't upgraded the API version at that time

https://github.com/hashicorp/terraform-provider-azurerm/blob/165f4b33f42e3948eaa53fa49316a48676b5d677/internal/services/compute/orchestrated_virtual_machine_scale_set.go#L68-L69

Can you please uncomment those lines and implement this when it's under the `windows_configuration` block?
```suggestion
```